ducksteroid/src/Timer.cs

```diff
@@ -6,53 +6,72 @@ namespace MyGame.src
     public class Timer
     {
         private int _lastcalled;
-        private int _min, _sec, _tick;
-        private Text t;
+        private Counter _min, _sec, _hr, _tick;
+        private Text _text;
 
         //Constructors
         public Timer ()
         {
-            _min = 2;
-            _sec = 30;
-            _tick = 58;
+            _tick = new Counter ();
+            _sec = new Counter ();
+            _min = new Counter ();
+            _hr = new Counter ();
             Font font = new Font ("arial.ttf", 20);
-            t = new Text (Color.White, "02:30", font);
+            _text = new Text (Color.White, "00:00", font);
             _lastcalled = TimeInSec;
         }
 
         //Properties
         public int LastCalledSec { get => _lastcalled; set => _lastcalled = value; }
-        public int TimeInSec { get { return _min * 60 + _sec; } }
-        public int Min { get => _min; }
-        public int Sec { get => _sec; }
+        public int TimeInSec { get { return _min.Value * 60 + _sec.Value; } }
+        private Counter Hr { get => _hr; }
```

Comment:
why do you need these private properties if you can just access them normally within the class

Reply:
It's more clear when using the filed.

Reply:
its not rlly necessary but if u say so